Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -245,6 +249,14 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
                 if (_leftCommandPanel != null)
                 {
                     _leftCommandTransform = _leftCommandPanel.RenderTransform as CompositeTransform;
+
+                    _leftCommandAnimation = new DoubleAnimation();
```

Comment:
Interesting idea :-). From what I can see it is not possible to cancel an animation in the animation library. And that will be an issue for instance if user passes above the threshold, which starts the animation, and then immediately goes below the threshold again. Then the animation needs to be cancelled and text and icon show be set on a fixed position. This is covered in the current PR, not sure how to do that with the library.

I’m also working with another tiny PR with another animation that uses clipping which I hopes gets accepted. And as I understand it this is not supported in the animation library. I think it’s a good idea all animation is created in the same way in this control.

That said, I’m willing to test this if you think this is a good idea :-).